src/components/activities/Activity.jsx

```diff
@@ -132,6 +132,7 @@ const TurnRevealWrapper = styled.div`
 
   .turnreveal-container {
     width: fit-content;
+    overflow: hidden;
```

Comment:
This seems to break shadows on the left and right of the turnreveal container,

maybe add something like
```css
padding: 5px;
margin: -5px;
```
This does make the content flip a little bit outside of the container, maybe there is a different solution?

Reply:
Thanks for noticing! 

I moved the shadow to the outer element, and made sure it tightly hugs the inner elements.
I had to override the shadow of one of the inner elements to make sure the shadows don't give weird animation artifacts.

Everything looks nice and good to me now, can you give it a last check?